KarhooUISDK/Workers/Formatters/KarhooDateFormatter.swift

```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 
+@available(*, deprecated, message: "The protocol will not be `public` from next SDK vesion")
```

Comment:
Does this affect Farechoice by any chance? Why was it public in the first place?

Reply:
There is one unused `DateFormatterType` invocation in Farechoice. It can be safely deleted. I guess it was planed to be used in ferachoice tests.